### ISSUE TITLE
Testing support?

### ISSUE THREAD

**Issue by skynyrd** [gothinkster/react-redux-realworld-example-app#41]
Hi, Thank you for the project, I really like the idea.
I use React + Redux for about a year however I don't feel very comfortable in testing.

Are tests going to be implemented?

Thanks

**Comment by EricSimons** [comment]
Great suggestion! @kentcdodds actually created some for this repo + the [node repo](https://github.com/gothinkster/node-express-realworld-example-app) over at https://github.com/kentcdodds/testing-workshop.

Maybe we could convince him to make a PR back here at some point :) 🙏

**Comment by skynyrd** [comment]
That would be great :)

**Comment by kentcdodds** [comment]
Perhaps! Though I did make a number of changes to make the code more testable. 🙃

**Comment by EricSimons** [comment]
In that case, we'd _love_ to have those changes PR'd 🙌

**Comment by kentcdodds** [comment]
I'm sure! It's unlikely I'll get around to it, but maybe someone who takes the workshop will be inspired :)

**Comment by justDanielMata** [comment]
@EricSimons I wrote some unit tests and changed a little bit of code to be more clean, i would love to write a full test suite for this, should i pr the initial tests so you see if they are okay before moving on?

**Comment by EricSimons** [comment]
@justDanielMata yes absolutely!! 🙌

**Comment by justDanielMata** [comment]
@EricSimons i'm probably dumb, but it is the first time i create a pull request for an oss project, how do i create the pr? it says i'm inauthorized on my terminal.

**Comment by EricSimons** [comment]
@justDanielMata github actually has a button in the UI that will automatically do that PR for you — where is your current forked repo located?

**Comment by justDanielMata** [comment]
Oh maybe that is the issue, i cloned the repo, didn't fork it, i'm trying
to submit a pr pushing from my terminal, should i fork it?

2017-06-06 18:18 GMT-03:00 Eric Simons <notifications@github.com>:

**Comment by EricSimons** [comment]
Yup! Heres some more info on how to do it: https://stackoverflow.com/questions/14680711/how-to-do-a-github-pull-request

**Comment by by12380** [comment]
Is the effort on adding tests still going on?
